multi-cluster-orchestrator/samples/hello_world/2-workload/charts/hello-world-application/templates/applicationset.yaml

```diff
@@ -18,31 +18,44 @@ metadata:
   name: hello-world-applicationset
   namespace: argocd
 spec:
-    generators:
-    - plugin:
-        configMapRef:
-          name: argocd-mco-placement
-        input:
+  generators:
+  - plugin:
+      configMapRef:
+        name: argocd-mco-placement
+      input:
+        parameters:
+          multiClusterOrchestraPlacementName: "hello-web-placement-autoscale"
+          multiClusterOrchestraPlacementNamespace: {{ .Release.Namespace }}
+      requeueAfterSeconds: 10
+  syncPolicy:
+    applicationsSync: sync
+    preserveResourcesOnDeletion: false
+  template:
+    metadata:
+      name: '{{`{{name}}`}}-hw'
+    spec:
+      destination:
+        namespace: {{ .Release.Namespace }}
+        name: '{{`{{name}}`}}'
+      syncPolicy:
+        automated: {}
+        syncOptions:
+          - CreateNamespace=true
+      project: default
+      source:
+        path: fleet-charts/hello-world
+        repoURL: https://github.com/GoogleCloudPlatform/gke-fleet-management.git
+        targetRevision: HEAD
+  templatePatch: |
+    spec:
+      source:
+        helm:
           parameters:
-            multiClusterOrchestraPlacementName: "hello-web-placement-autoscale"
-            multiClusterOrchestraPlacementNamespace: {{ .Release.Namespace }}
-        requeueAfterSeconds: 10
-    syncPolicy:
-      applicationsSync: sync
-      preserveResourcesOnDeletion: false
-    template:
-      metadata:
-        name: '{{`{{name}}`}}-hw'
-      spec:
-        destination:
-          namespace: {{ .Release.Namespace }}
-          name: '{{`{{name}}`}}'
-        syncPolicy:
-          automated: {}
-          syncOptions:
-            - CreateNamespace=true
-        project: default
-        source:
-          path: fleet-charts/hello-world
-          repoURL: https://github.com/GoogleCloudPlatform/gke-fleet-management.git
-          targetRevision: HEAD
+            - name: hf_api_token
```

Comment:
We don't need HF token for hello-world app right?

Reply:
You are correct, I will fix!  Accidentally got included in the copy/paste of the replicas parameters.